test/test_xfile.py

```diff
@@ -28,24 +27,24 @@ def assertContainsUserNamespace(self, filename):
                 doc_beginning_excerpt += (in_file.readline().strip())  # first line should be xml tag
                 lineCount += 1
             found = doc_beginning_excerpt.rfind("xmlns:user=")
-            #print(doc_beginning_excerpt[found:found+10])
+            # print(doc_beginning_excerpt[found:found+10])
             self.assertRegex(doc_beginning_excerpt, "xmlns:user=")
-                
+
     def test_save_preserves_namespace_twb(self):
-        filename = COMPLEX_TWB 
+        filename = COMPLEX_TWB
         self.assertContainsUserNamespace(filename)
         wb = Workbook(filename)
         new_name = 'saved-as-twb.twb'
         wb.save_as(new_name)
         self.assertContainsUserNamespace(new_name)
-'''   
 
-'''            
+
+'''
     def demo_bug_ns_not_preserved_if_not_used(self):
         filename = TABLEAU_10_TDS
         self.assertContainsUserNamespace(filename)
         wb = Datasource.from_file(filename)
-        #wb.save() 
+        #wb.save()
```

Comment:
Why did we add this comment? I hope not a typo.

Reply:
If you mean this line, I was checking variations of save and save-as to make sure they worked the same. It can be removed altogether.

The test case as a whole documents an edge case I found which wasn't worth delaying the main fix for, but is a theoretical bug in the future. The test case is kept as documentation, but commented out so the pipelines succeed.

Reply:
Ok thanks, then some more comments in this function would be nice.